README.md

```diff
@@ -86,8 +86,9 @@ components in the Pulsar ecosystem, including connectors, adapters, and other la
 ## Build Pulsar
 
 Requirements:
- * Java JDK 1.8 or Java JDK 11
- * Maven 3.3.9+
+ * Java 8 JDK (for building Pulsar)
+   * When building on newer than Java 8, the resulting artifacts are not compatible with Java 8 runtime because of issues such as https://github.com/apache/pulsar/issues/8445.
```

Comment:
```suggestion
   * When building Pulsar on a higher version (higher than Java 8), the resulting artifacts are not compatible with Java 8 runtime because of some issues, such as [issue 8445](https://github.com/apache/pulsar/issues/8445).
```